pkg/controller/master/backup/backup_metadata.go

```diff
@@ -132,18 +133,40 @@ func (h *MetadataHandler) syncVMBackup(target *settings.BackupTarget) error {
 		return err
 	}
 
+	var namespaceFolders []string
+	var requiredMovingFilePaths []string
 	for _, fileName := range fileNames {
-		backupMetadata, err := loadBackupMetadataInBackupTarget(filepath.Join(metadataFolderPath, fileName), bsDriver)
+		filePath := filepath.Join(metadataFolderPath, fileName)
+		if bsDriver.FileExists(filePath) {
+			requiredMovingFilePaths = append(requiredMovingFilePaths, filePath)
+		} else {
+			namespaceFolders = append(namespaceFolders, filePath)
+		}
+	}
```

Comment:
Just a Nit, how about following for using less brackets?
```
	for _, fileName := range fileNames {
		filePath := filepath.Join(metadataFolderPath, fileName)
		if bsDriver.FileExists(filePath) {
			requiredMovingFilePaths = append(requiredMovingFilePaths, filePath)
                        continue;
		} 
		namespaceFolders = append(namespaceFolders, filePath)
	}
```

Reply:
Updated it. Thanks.